lib/utils/utils.go

```diff
@@ -682,6 +682,11 @@ func ByteCount(b int64) string {
 // ErrLimitReached means that the read limit is reached.
 var ErrLimitReached = &trace.LimitExceededError{Message: "the read limit is reached"}
 
+// ErrClientCredentialsHaveExpired means
+// that the credentials expired on the server-side,
+// and the user should relogin.
```

Comment:
nit: Use longer lines?

```suggestion
// ErrClientCredentialsHaveExpired means that the credentials expired on the
// server-side, and the user should relogin.
```